### ISSUE TITLE
Minitest::Reporters::DefaultReporter reporting full path - not relative

### ISSUE THREAD

**Issue by kwerle** [minitest-reporters/minitest-reporters#329]
I have not poked at the code, but in my project Minitest::Reporters::DefaultReporter is returning the full path, not relative.

I switched to junit just to check, and it is doing the right thing.
https://issueantenna.com/repo/minitest-reporters/minitest-reporters/issues/305

**Comment by Earlopain** [comment]
I agree that there should at least be an option for relative paths. My application is in docker and emitting absolute paths prevents VSCode from automatically jumping to the file when clicking on the paths. I have to remove the stuff before the working directory myself.
